uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java

```diff
@@ -0,0 +1,39 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.browser;
+
+import com.microsoft.identity.client.ui.automation.app.IApp;
+
+/**
+ * An interface describing a browser app on an Android device during UI Automated test.
+ */
+public interface IBrowser extends IApp {
+
+    /**
+     * Browse to the supplied url using this browser.
+     *
+     * @param url the url to open
+     */
+    void browse(String url);
```

Comment:
Suggestion: `navigateTo()` (`browse()` wasn't very to me)

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/982/commits/172b12c97b3ea1654257c7f96a20ffb06447a529